### ISSUE TITLE
Fine-tuning details

### ISSUE THREAD

**Issue by xfgao** [google/spiqa#7]
Thanks for the great work. 

A quick question: when you finetuned the open-sourced models, in addition to the reference image and question, were you providing the image caption as input to the model? 

In addition to the answer, are you using the explanation as part of the ground-truth response?

**Comment by ShramanPramanick** [comment]
Sorry for the late response. We used the figure/table captions during fine-tuning InstructBLIP-7B and LLaVA-1.5-7B. Given the reference image, caption, and question, the model is expected to generate the answer.

We do not use the explanations as ground-truth responses; only the answer is used for calculating loss.

**Comment by vsubhashini** [comment]
Thanks, resolving this.
